main.h

```diff
@@ -0,0 +1,11 @@
+#ifndef MAIN_H
+#define MAIN_H
+
+#include <stdio.h>
```

Comment:
are you sure we are allowed to use the standard input and output library

Reply:
i think because the project is printf and we use standard input and output for printf, i just decided to add it just incase.

Reply:
> i think because the project is printf and we use standard input and output for printf, i just decided to add it just incase.

Okay, good idea. We could use it in testing stuff out.

Reply:
yes